src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -704,6 +706,38 @@ public function exportAction(CustomerFilters $filters)
             ->setFileName('customer_' . date('Y-m-d_His') . '.csv');
     }
 
+    /**
+     * @AdminSecurity("is_granted(['read'], request.get('_legacy_controller'))")
```

Comment:
We'll have to check what happens if you have the rights to Order page but no rights to Customer page. Best behavior would be a nice error message 🤔 .

However, to be logical we should secure this action with "you need EITHER the customer READ rights or the order READ rights"